### PR TITLE
Hiring site updates

### PR DESCRIPTION
**Pull request by ctro**
1. Add a consulting engineer section and initial question
2. Add a couple questions that had been suggested and added in the master GDoc
3. Fix some broken links reported in #23

### CODE REVIEW

**Comment by fureigh** [comment]
Thanks for doing this, @ctro! I'll review this afternoon.

File: _pages/interviews/consulting-engineer.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Consulting Engineer Interview Guide
+parent: interviews
+---
+
+*This is a supplement to the [technical engineering interview guide]({{ site.baseurl }}/interviews/engineer/). Please read that guide first. The questions here can be added to the technical interview for Consulting Engineer candidates.*
+
+### Consulting Engineer Questions
+
+* [“Please explain a complex technical concept or tool to me as if I were a non-technical partner](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.8lizmty3jisl)
```
Comment (ertzeid) [inline _pages/interviews/consulting-engineer.md]: I really like this.

Comment (ctro) [reply _pages/interviews/consulting-engineer.md]: Hat tip to @mgwalker thanks again for the suggestion

Comment (fureigh) [reply _pages/interviews/consulting-engineer.md]: This is delightful. Minor copy edit: remove the open-quote from the link text for consistency with the other questions (`"Please explain` --> `Please explain`).

File: _pages/interviews/core-values.md [hunk 2]
```diff
@@ -119,8 +121,9 @@ Regardless of whether or not the candidate is applying for a software developmen
     - If applicable: Were you successful in doing so, and why or why not?
 
 - [What is your comfort level with peer-reviewed work?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
+    - If applicable: Have you pro-actively sought additional training and/or peer review?
```
Comment (fureigh) [inline _pages/interviews/core-values.md]: Minor copy edit: `pro-actively` --> `proactively`

File: _pages/resume-review.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Before you begin a review session, [remind yourself to watch out for unconscious
 
 Each position comes with a *crediting plan* -- a formal resume review rubric that we use to "score" resumes and separate applications into a set that'll move on to interviews, and a set we'll reject.
 
-See [the list of crediting plans]({{ site.baseurl }}/crediting-plans/) for links to all the ones we have for engineering roles, and for convenient single-page checklists you can print out and use while reviewing resumes if you like.
+See [the list of crediting plans]({{ site.baseurl }}/resume-review/crediting-plans/) for links to all the ones we have for engineering roles, and for convenient single-page checklists you can print out and use while reviewing resumes if you like.
```
Comment (fureigh) [inline _pages/resume-review.md]: Thanks for correcting this link!

File: _pages/interviews/core-values.md (hunk 2 above)
Comment (ctro) [reply _pages/interviews/core-values.md]: :+1: fixed

File: _pages/interviews/consulting-engineer.md (hunk 1 above)
Comment (ctro) [reply _pages/interviews/consulting-engineer.md]: Removed `"` . Thanks!

File: _pages/resume-review.md (hunk 3 above)
Comment (ctro) [reply _pages/resume-review.md]: 🎉

**Comment by ctro** [comment]
@fureigh ready for another review pass when you are, thank you!

**Pull request merged**